pkgs/by-name/op/opendht/package.nix

```diff
@@ -30,6 +31,17 @@ stdenv.mkDerivation {
     hash = "sha256-qErVKyZQR/asJ8qr0sRDaXZ8jUV7RaSLnJka5baWa7Q=";
   };
 
+  patches = [
+    (fetchpatch2 {
+      url = "https://github.com/savoirfairelinux/opendht/commit/2c26030108ede63e74efcc2d9c23ad47fdc8372e.patch?full_index=1";
```

Comment:
This patch fails to apply on my end

Reply:
That's… weird, I don't see any failure on my end

Reply:
It sounds like the plan should be to land https://github.com/NixOS/nixpkgs/pull/410596, then https://github.com/NixOS/nixpkgs/pull/399234, then I can update this PR once it has reached staging

Reply:
Yes, that's a good plan.

I'm going to merge #410596 in 1-2 days (giving the maintainer some time to review). By the way, the master → staging merge can be done on demand by committers too.

Reply:
Done. The other PRs should be in staging now, so this can be rebased.